### PR TITLE
ISPN-10713 2.2.4.RELEASE and 10.1.1.Final

### PR DESCRIPTION
**Pull request by karesti**
* Upgrade to Infinispan 10.1.1.Final
* SB 2.2.4.RELEASE

https://issues.jboss.org/browse/ISPN-10713

### CODE REVIEW

**Comment by tristantarrant** [comment]
@karesti we will want a release based on 9.4.x with SB 2.2.

**Comment by karesti** [comment]
@tristantarrant when

**Comment by karesti** [comment]
@tristantarrant this week ?

**Comment by vincent-petit** [comment]
Hi there,
What is the state of this pull request?
The support of Infinispan 10.0.X in infinispan-spring-boot is something we would really like

**Comment by karesti** [comment]
@vincent-petit a couple of minor things to be done and then the release will be coming

File: infinispan-spring-boot-samples/remote/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.infinispan</groupId>
         <artifactId>infinispan-spring-boot-starter-parent</artifactId>
-        <version>2.1.8-SNAPSHOT</version>
+        <version>2.2.0-SNAPSHOT</version>
```
Comment (aureamunoz) [inline infinispan-spring-boot-samples/remote/pom.xml]: The master branch in `infinispan-spring-boot-starter-parent' is still pointing to `2.1.8`, right? 
https://github.com/infinispan/infinispan-spring-boot/blob/115d6a6b7dfab0b523610d107985feaf8f69a0e9/pom.xml#L13

Comment (karesti) [reply infinispan-spring-boot-samples/remote/pom.xml]: master yes. this is for the new release 2.2.x

File: infinispan-spring-boot-starter-remote/pom.xml [hunk 2]
```diff
@@ -73,6 +79,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.10.0</version>
```
Comment (pruivo) [inline infinispan-spring-boot-starter-remote/pom.xml]: can you use `${version.jackson.databind}`? 
So I don't have to worry about it downstream :)

Comment (karesti) [reply infinispan-spring-boot-starter-remote/pom.xml]: yes

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**